src/components/Header/UserName.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useQuery } from 'react-query';
+import { Flex, Text, useColorModeValue } from '@chakra-ui/react';
+
```

Comment:
remove extra space

Reply:
It's from eslint: "simple-import-sort/imports"
<img width="475" alt="eslint" src="https://user-images.githubusercontent.com/80061297/182048693-ea9552a3-a888-4d4e-9fe2-a98561ad57b3.PNG">